allensdk/brain_observatory/behavior/data_objects/metadata/ophys_experiment_metadata/multi_plane_metadata/multi_plane_metadata.py

```diff
@@ -55,9 +43,10 @@ def from_lims(
         return cls(
             ophys_experiment_id=ophys_experiment_metadata.ophys_experiment_id,
             ophys_session_id=ophys_experiment_metadata._ophys_session_id,
-            ophys_container_id=ophys_experiment_metadata._ophys_container_id,     # noqa E501
+            ophys_container_id=ophys_experiment_metadata._ophys_container_id,
             field_of_view_shape=ophys_experiment_metadata._field_of_view_shape,
             imaging_depth=ophys_experiment_metadata._imaging_depth,
+            average_container_depth=ophys_experiment_metadata._average_container_depth,  # noqa E501
```

Comment:
Does black really not split this line and the others like it?

Reply:
There's been a ticket open for this on the black repository since 2018 and it hasn't been resolved yet.
https://github.com/psf/black/issues/510

A black collaborator acknowledged the issue needs to be fixed in 2020
https://github.com/psf/black/issues/1582#issuecomment-672009696

Reply:
huh, if you fix it by hand, does black just revert it?

Reply:
Yes, it gets reverted.

Reply:
okay, guess we'll live with it.